components/card.css

```diff
@@ -0,0 +1,61 @@
+.card {
```

Comment:
zum Dateinamen: "card" ist sehr allgemein, vielleicht wäre "quiz-card" aussagekräftiger, wie in Merles Beispiel.. Wenn du dann irgendwann z.B. eine Visitenkarte hast, dann kann man die einzelnen .css Files evtl nicht mehr ganz so gut zuordnen...